functional-tests/step_definitions/custom_test_steps.js

```diff
@@ -0,0 +1,28 @@
+const methods_interactions = require('kakunin').methods.interactions;
+const methods_checkers = require('kakunin').methods.checkers;
+
+const { When, Then } = require('kakunin');
+
+When(/^I click the button$/, function() {
+    const elementName = element(by.xpath('//button[@id="button"]'));
+    return methods_interactions.click(this.currentPage, elementName);
+});
+
+Then(/^there are "([^"]*)" elements$/, function(numberExpression) {
+    const elementName = $$('table tr');
+    return methods_checkers.checkNumberOfElements(this.currentPage, numberExpression, elementName);
+});
+
+Then(/^the button is not visible$/, function() {
+    const elementName = element(by.xpath('//button[@id="button"]'));
```

Comment:
it's `element`

Reply:
I changed this for "elem". "element" can't be used in this case because it's duplicate with element from protractor what cause issues

Reply:
So how about we name it what it is - in this case it's  button